src/main/java/org/apache/commons/jxpath/ri/JXPathContextReferenceImpl.java

```diff
@@ -27,23 +27,10 @@
 import java.util.Vector;
 import java.util.Map.Entry;
 
-import org.apache.commons.jxpath.CompiledExpression;
-import org.apache.commons.jxpath.ExceptionHandler;
-import org.apache.commons.jxpath.Function;
-import org.apache.commons.jxpath.Functions;
-import org.apache.commons.jxpath.JXPathContext;
-import org.apache.commons.jxpath.JXPathException;
-import org.apache.commons.jxpath.JXPathFunctionNotFoundException;
-import org.apache.commons.jxpath.JXPathInvalidSyntaxException;
-import org.apache.commons.jxpath.JXPathNotFoundException;
-import org.apache.commons.jxpath.JXPathTypeConversionException;
-import org.apache.commons.jxpath.Pointer;
+import org.apache.commons.jxpath.*;
```

Comment:
The project does not appear to use wildcard imports, let's stick with existing conventions for now.

Reply:
Changed.